### PR TITLE
Fix typos in 36 files | Almost only documentation

### PR DESCRIPTION
**Pull request by Sajjon**
Fix typos in 36 files

### CODE REVIEW

File: contrib/tracing/p2p_monitor.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def loop(screen, bpf, peers):
     info_panel = panel.new_panel(win)
     info_panel.hide()
 
-    ROWS_AVALIABLE_FOR_LIST = curses.LINES - 5
+    ROWS_AVAILABLE_FOR_LIST = curses.LINES - 5
```
Comment (Sajjon) [inline contrib/tracing/p2p_monitor.py:173]: `rg "ROWS_AVALIABLE_FOR_LIST"` shows empty, so all relevant references have been renamed.

File: doc/Doxyfile.in [hunk 2]
```diff
@@ -1485,7 +1485,7 @@ EXT_LINKS_IN_WINDOW    = NO
 
 FORMULA_FONTSIZE       = 10
 
-# Use the FORMULA_TRANPARENT tag to determine whether or not the images
+# Use the FORMULA_TRANSPARENT tag to determine whether or not the images
```
Comment (Sajjon) [inline doc/Doxyfile.in:1488]: The config *is* named `FORMULA_TRANSPARENT` so this is purely a typo correction, no logic effect.

File: src/qt/clientmodel.cpp [hunk 3]
```diff
@@ -260,7 +260,7 @@ void ClientModel::subscribeToCoreSignals()
         }));
     m_event_handlers.emplace_back(m_node.handleBannedListChanged(
         [this]() {
-            qDebug() << "ClienModel: Requesting update for peer banlist";
+            qDebug() << "ClientModel: Requesting update for peer banlist";
```
Comment (Sajjon) [inline src/qt/clientmodel.cpp:263]: Seems to only be a debug print, so should be ok?

File: test/functional/wallet_backup.py [hunk 4]
```diff
@@ -116,7 +116,7 @@ def erase_three(self):
     def restore_invalid_wallet(self):
         node = self.nodes[3]
         invalid_wallet_file = self.nodes[0].datadir_path / 'invalid_wallet_file.bak'
-        open(invalid_wallet_file, 'a', encoding="utf8").write('invald wallet')
+        open(invalid_wallet_file, 'a', encoding="utf8").write('invalid wallet')
```
Comment (Sajjon) [inline test/functional/wallet_backup.py:119]: This is just a test, so should be OK to correct typo in the written contents to file test file?

**Comment by fanquake** [comment]
Thanks, however most of these are in upstream code (subtrees, macros, Doxygen config), or code we aren't going to change (like the release notes). You'll need to remove all of those changes, and then also write a proper commit message.

**Comment by Sajjon** [comment]
> Thanks, however most of these are in upstream code (subtrees, macros, Doxygen config), or code we aren't going to change (like the release notes). You'll need to remove all of those changes, and then also write a proper commit message.

👍 thx! Will give it another go later tonight. Any suggsetion on what a "proper commit message" would be?

**Comment by fanquake** [comment]
> Any suggestion on what a "proper commit message" would be?

https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md.

**Review by glozow — COMMENTED** [review]
Thanks for your interest in contributing. Maybe take a look at https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#pull-request-philosophy.

Fwiw I think editing archived release notes (especially changing the names of pull requests) just makes them inaccurate. I think this should be closed.

**Comment by stickies-v** [comment]
> I think this should be closed.

Agreed. Concept NACK.

**Pull request closed without merging**